features/admin.users.v1/pages/users.tsx

```diff
@@ -959,6 +1038,21 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
         mutateParentOrgUserInviteList();
     };
 
+    /**
+     * Handles removing a account status filter.
+     *
+     * @param filterKey - The key of the filter to remove
+     */
+    const handleAccountStatusFilterRemove = (filterKey: string): void => {
+        setSelectedAccountStatusFilters(
+            (statusFilterKeys: string[]) =>
+                statusFilterKeys.filter(
+                    (statusFilterKey: string) =>
+                        statusFilterKey !== filterKey
+                ));
+        setListOffset(1);
```

Comment:
could you explain the purpose of this state change?

Reply:
This is to reset the pagination